frontend/src/core/entities/actions.js

```diff
@@ -102,6 +117,18 @@ export function get(
     };
 }
 
+export function getSiblingEntities(entity: number, locale: string): Function {
+    return async (dispatch) => {
+        dispatch(request());
```

Comment:
You can remove this line - it's only needed when loading entities for the entire entity list.